format.go

```diff
@@ -79,8 +88,49 @@ func (upErr *UnpackedError) ToString(format Format) string {
 	return str
 }
 
-// ToJSON returns a JSON formatted map for a given eris error.
-func (upErr *UnpackedError) ToJSON(format Format) map[string]interface{} {
+// ToString returns a default formatted string for a given eris error.
+//
+// An error without trace will be formatted as <Root error msg>:<Wrap error msg> for the default format.
+// An error with trace will be formatted as following for the default format.
+// <Root error msg>\n
+// \t<Method1>:<File1>:<Line1>\n
```

Comment:
change ":" to something like "[Format. StackElemSep]"

Reply:
for default format, I decided to go with the resolved values rather than the object for clarity, is it not clear?

Reply:
oh, didn't realize this is for default. this is fine as is